graalwasm/graalwasm-micronaut-excelice/README.md

```diff
@@ -0,0 +1,32 @@
+# Excelize with GraalWasm Micronaut Demo
+
+This demo illustrates how GraalWasm can be used to embed Excelize , a WebAssembly Go library for creating and reading Microsoft Excel (xlsx) spreadsheets.
+The demo also uses GraalJS to access the Excelize module through the WebAssembly JavaScript API.
+
+## Preparation
+
+Install GraalVM for JDK 21 and set the value of `JAVA_HOME` accordingly.
+We recommend using [SDKMAN!](https://sdkman.io/). (For other download options, see [GraalVM Downloads](https://www.graalvm.org/downloads/).)
+
+```bash
+sdk install java 21-graal
```

Comment:
```suggestion
Install GraalVM for JDK 24 and set the value of `JAVA_HOME` accordingly.
We recommend using [SDKMAN!](https://sdkman.io/). (For other download options, see [GraalVM Downloads](https://www.graalvm.org/downloads/).)

```bash
sdk install java 24-graal
```
```suggestion
Install GraalVM for JDK 21 and set the value of `JAVA_HOME` accordingly.
We recommend using [SDKMAN!](https://sdkman.io/). (For other download options, see [GraalVM Downloads](https://www.graalvm.org/downloads/).)

```bash
sdk install java 21-graal
```